functorch/_src/aot_autograd.py

```diff
@@ -1166,11 +1184,19 @@ def wrapped_compiled_fn(*args):
     return maybe_wrap_debug(wrapped_compiled_fn)
 
 
+def describe_input(i, aot_config):
+    if i < aot_config.num_params_buffers:
+        return f"parameter/buffer {i}"
```

Comment:
not necessary, but would be even nicer if we eventually mapped this back to the param/buffer name

Reply:
yeah, it's pretty easy, i just didn't do it lol